edge-modules/api-proxy-module/build.sh

```diff
@@ -88,7 +88,7 @@ echo ${PROJECT_ROOT}
 
 if [[ "$ARCH" == "amd64" ]]; then
 set +e
-../../scripts/linux/cross-platform-rust-build.sh --os alpine --arch $ARCH --build-path edge-modules/api-proxy-module
+docker run --rm -it -v "${PROJECT_ROOT}":/home/rust/src ekidd/rust-musl-builder cargo build --release --manifest-path /home/rust/src/edge-modules/api-proxy-module/Cargo.toml
```

Comment:
I am fine with this.. but any reason why we cannot use the script?

Reply:
We can use it, I can roll it back. It's just this takes a few minutes while the script takes 20mn. That's the only reason.

Reply:
Do we need to trim the script? 3 min vs 20 min is a huge difference... what all else is the script doing?

Reply:
My changes makes it last 3mn :). It is doing the same thing. It is pulling pre compiled image from git hub instead of building the image